npc/custom/jobmaster.txt

```diff
@@ -292,15 +355,105 @@ Get_Platinum:
 		break;
 	}
 	return;
+}
+
+// Function which gives a job related item to the player
+// the items are the rewards from the original job change quests
```

Comment:
`I would prefer if it would use eaclass() with bitmasking instead of hardcoding all job versions for each class.`
Stolao already suggested to use `BaseClass` and `BaseJob`.
`Also I think some might need a renewal check?`
Yeah it would be great if someone could check the jobchange quest items in detail. Currently it's more like a semi official feature to improve the game flow. (But honestly the whole NPC is non official. So it shouldn't be a big deal if the player gets an Blade instead of a Falcion from a non official NPC in an option which is disabled by default.)